testdata/picard/vcf/allHomRef.vcf

```diff
@@ -0,0 +1,147 @@
+##contig=<ID=GL000241.1,length=42152>
+##contig=<ID=GL000243.1,length=43341>
+##contig=<ID=GL000242.1,length=43523>
+##contig=<ID=GL000230.1,length=43691>
+##contig=<ID=GL000237.1,length=45867>
+##contig=<ID=GL000233.1,length=45941>
+##contig=<ID=GL000204.1,length=81310>
+##contig=<ID=GL000198.1,length=90085>
+##contig=<ID=GL000208.1,length=92689>
+##contig=<ID=GL000191.1,length=106433>
+##contig=<ID=GL000227.1,length=128374>
+##contig=<ID=GL000228.1,length=129120>
+##contig=<ID=GL000214.1,length=137718>
+##contig=<ID=GL000221.1,length=155397>
+##contig=<ID=GL000209.1,length=159169>
+##contig=<ID=GL000218.1,length=161147>
+##contig=<ID=GL000220.1,length=161802>
+##contig=<ID=GL000213.1,length=164239>
+##contig=<ID=GL000211.1,length=166566>
+##contig=<ID=GL000199.1,length=169874>
+##contig=<ID=GL000217.1,length=172149>
+##contig=<ID=GL000216.1,length=172294>
+##contig=<ID=GL000215.1,length=172545>
+##contig=<ID=GL000205.1,length=174588>
+##contig=<ID=GL000219.1,length=179198>
+##contig=<ID=GL000224.1,length=179693>
+##contig=<ID=GL000223.1,length=180455>
+##contig=<ID=GL000195.1,length=182896>
+##contig=<ID=GL000212.1,length=186858>
+##contig=<ID=GL000222.1,length=186861>
+##contig=<ID=GL000200.1,length=187035>
+##contig=<ID=GL000193.1,length=189789>
+##contig=<ID=GL000194.1,length=191469>
+##contig=<ID=GL000225.1,length=211173>
+##contig=<ID=GL000192.1,length=547496>
+##contig=<ID=NC_007605,length=171823>
+##reference=file:///seq/references/Homo_sapiens_assembly19/v1/Homo_sapiens_assembly19.fasta
+#CHROM	POS	ID	REF	ALT	QUAL	FILTER	INFO	FORMAT	HG00116	HG00123	HG00158	HG00160	HG00265	HG00311	HG00371	HG00380	HG00404	HG00551	HG00610	HG00628	HG00635	HG01048	HG01079	HG01094	HG01247	HG01256	HG01260	HG01461	HG01488	NA06989	NA11918	NA11919	NA12341	NA18510	NA18520	NA18522	NA18574	NA18634	NA18867	NA18960	NA18986	NA19004	NA19067	NA19092	NA19102	NA19474	NA19675	NA19703	NA19711	NA19725	NA19819	NA19908	NA19920	NA20787	NA20798	NA20801	NA20803	NA20805
+1	69270	.	A	G	569.98	PASS	AC=23;AF=0.767;AN=30;BaseQRankSum=-2.920;DP=4121;Dels=0.00;FS=0.000;HaplotypeScore=0.0000;InbreedingCoeff=0.3609;MLEAC=24;MLEAF=0.800;MQ=3.91;MQ0=1518;MQRankSum=0.523;QD=0.55;ReadPosRankSum=-0.861;SNPEFF_AMINO_ACID_CHANGE=S108;SNPEFF_CODON_CHANGE=tcA/tcG;SNPEFF_EFFECT=SYNONYMOUS_CODING;SNPEFF_EXON_ID=exon_1_69037_69829;SNPEFF_FUNCTIONAL_CLASS=SILENT;SNPEFF_GENE_BIOTYPE=protein_coding;SNPEFF_GENE_NAME=OR4F5;SNPEFF_IMPACT=LOW;SNPEFF_TRANSCRIPT_ID=ENST00000534990;VQSLOD=-8.590e+00;culprit=MQ	GT:AD:DP:GQ:PL	0/0:60,18:75:9:0,9,89	./.	./.	./.	./.	./.	./.	./.	./.	./.	1/1:51,23:71:3:24,3,0	./.	./.	./.	1/1:59,15:72:3:33,3,0	./.	./.	./.	./.	./.	0/0:2,0:2:6:0,6,45	0/0:66,8:71:3:0,3,24	./.	./.	./.	0/0:75,0:75:9:0,9,81	1/1:183,17:197:9:82,9,0	0/1:56,19:63:18:18,0,18	./.	1/1:22,22:43:3:31,3,0	./.	./.	1/1:51,24:73:6:57,6,0	1/1:52,25:74:12:101,12,0	./.	./.	./.	1/1:63,12:72:6:47,6,0	./.	./.	./.	./.	./.	./.	./.	./.	1/1:40,56:93:9:78,9,0	1/1:63,14:74:6:47,6,0	1/1:79,11:87:3:24,3,0	./.
```

Comment:
This comment is about the index file that you include.  It is better to use `VcfTestUtils::createTemporaryIndexedVcfFromInput` and then not include the idx in the repository.